pages/formulaire/merci.tsx

```diff
@@ -15,6 +17,13 @@ const ThanksPage: NextPageWithLayout = () => {
           <ButtonLink to="/">Retourner au moteur de recherche</ButtonLink>
         </div>
       </main>
+      <script
+        dangerouslySetInnerHTML={{
+          __html: `
+          var _paq = window._paq || [];
+          _paq.push(['setCustomDimension', '1', "${usager}"]);`,
+        }}
+      />
```

Comment:
I dont see the need for this. If user see this page shouldn't it have gone through `_document` first ?

Reply:
As for the other types(non-agents), they are not our focus right now so I would not add this only for them

Reply:
Yes, I made this change to start collecting behavior from other user types, for us to have informed data when we decide to improve UX for them as well (instead of having to wait for a specific research campaign). But I'm also understanding the « keep it minimum » approach when it comes to user tracking, so I can remove this.